lib/tether/tether.go

```diff
@@ -252,6 +257,41 @@ func (t *tether) setMounts() error {
 			return fmt.Errorf("unsupported volume mount type for %s: %s", k, v.Source.Scheme)
 		}
 	}
+	return t.populateVolumes()
```

Comment:
This works fine because we bind mount the parent directory and not the actual target. Bind mounting the parent directory results in mounting the original dir and not the mounted device 

https://superuser.com/a/389262/486095